cost/azure/savings_realized/azure_savings_realized.pt

```diff
@@ -473,7 +473,9 @@ script "js_get_aggregated_costs", type: "javascript" do
     return { "dimension": "bill_source", "type": "equal", "value": bill_source }
   })
 
-  filter["expressions"].push({ "type": "or", "expressions": bill_source_expressions })
+  if (bill_source_expressions.length  > 0) {
```

Comment:
There should be one space, not two, between `bill_source_expressions.length` and `>`

Reply:
Fixed.